Combinatorial_map/test/Combinatorial_map/Combinatorial_map_deprecated_test.cpp

```diff
@@ -0,0 +1,73 @@
+#define CGAL_NO_DEPRECATION_WARNINGS 1
+
```

Comment:
There should be something like:
```C++
#ifndef CGAL_NO_DEPRECATED_CODE
// the current content of the file
#else
int main() {}
#endif
```
otherwise that test will "fail" in test platforms where `CGAL_NO_DEPRECATED_CODE` is defined.